packages/core/src/Button/Button.spec.js

```diff
@@ -212,7 +212,6 @@ describe('Button', () => {
 
         expect(button).toHaveStyleRule('color', theme.palette.primary.base)
         expect(button).toHaveStyleRule('vertical-align', 'inherit')
-        expect(button).toHaveStyleRule('font-weight', '500')
```

Comment:
Should we still include a check for the correct `font-weight`?

Reply:
Couldn't hurt - currently it's undefined though since it's not on the underlying components anymore. Would we want to add it as a prop?

Reply:
That's what I was wondering about, I'm not sure where the font-weight actually originates. @sdalonzo thoughts?